### ISSUE TITLE
Radio buttons appearing as squares on iOS 9.1

### ISSUE THREAD

**Issue by tekoyaki** [twbs/bootstrap#20214]
Is this an intentional styling of bootstrap?

![screen shot 2016-06-30 at 1 39 21 pm](https://cloud.githubusercontent.com/assets/444671/16503913/1bb0076e-3eca-11e6-99fb-83cdd0e4e511.png)

Screenshot taken directly from BS4 docs page without alterations. Tested on iPhone 6s and iPhone simulator from xcode.

**Comment by cvrebert** [comment]
No, not intentional.

**Comment by tekoyaki** [comment]
It seems to be caused by input { border-radius: 0; }

Mobile safari uses border-radius: 8px by default.

**Comment by tekoyaki** [comment]
Pretty odd behavior of mobile Safari.

![simulator screen shot jul 4 2016 11 18 57 am](https://cloud.githubusercontent.com/assets/444671/16567962/660acbd4-41d9-11e6-9c75-ee74efe38b49.png)

http://codepen.io/radiuskuntoro/pen/rLwzpE

**Comment by cvrebert** [comment]
Indeed. Using the style inspector shows that iOS Safari's UA stylesheet includes:
```css
input[type="radio"] {
  -webkit-appearance: radio;
  border-top-left-radius: 8px;
  border-top-right-radius: 8px;
  border-bottom-right-radius: 8px;
  border-bottom-left-radius: 8px;
  width: 16px;
  height: 16px;
  padding-top: 0px;
  padding-right: 0px;
  padding-bottom: 0px;
  padding-left: 0px;
  background-color: rgba(255, 255, 255, 0.00784314);
}
```

And as your CodePen demonstrates, our code is also affecting iOS Safari's default checkbox styles, which are:
```css
input[type="checkbox"] {
  -webkit-appearance: checkbox;
  border-top-left-radius: 5px;
  border-top-right-radius: 5px;
  border-bottom-right-radius: 5px;
  border-bottom-left-radius: 5px;
  width: 16px;
  height: 16px;
  padding-top: 0px;
  padding-right: 0px;
  padding-bottom: 0px;
  padding-left: 0px;
  background-color: rgba(255, 255, 255, 0.00784314);
}
```

**Comment by mdo** [comment]
Fixed by #20695.
